.github/pull_request_template.md

```diff
@@ -6,6 +6,20 @@
 
 (Details)
 
+## Does this PR affect the documentation?
+
+- [ ] No. You can skip the rest of this section.
+- [ ] Yes. Make sure the affected pages / sections render correctly by following the steps below.
+
+Steps:
+
+1. Check the status of the `ci/circleci: build_doc` check. If it's successful, proceed to the
```

Comment:
Does this test need to be manually started, or does it always run, or is it automatically triggered whenever a docs change is detected?

Reply:
It's always automatically triggered!